server/makefile

```diff
@@ -5,7 +5,7 @@ CFLAGS	= -g -O3 -Wall -std=c++11 -pthread -D _DEBUG_
 LDFLAGS	= -lavcodec -lavformat -lmysqlcppconn -lpulse-simple -lpulse -lrt -lprotobuf
 
 SRC		= server.cpp logger.cpp connection_manager.cpp network_component.cpp \
-		  db_component.cpp track.cpp audio_component.cpp \
+		  db_component.cpp track.cpp audio_component.cpp discover_dacs.cpp \
```

Comment:
Might be good to list one `cpp` file per line for easier addition/removal. Up to you though.